frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -748,6 +761,48 @@ const ManagePolicyPage = ({
     }
   };
 
+  const onUpdateConditionalAccess = async ({
+    enabled: enableConditionalAccess,
+  }: IConditionalAccessFormData) => {
+    setIsUpdatingPolicies(true);
+
+    try {
+      if (teamIdForApi === API_NO_TEAM_ID) {
+        // patch global config (No team case)
+        const payload = {
+          integrations: {
+            ...globalConfig?.integrations,
+            conditional_access_enabled: enableConditionalAccess,
+          },
+        };
+        // TODO - confirm nothing getting unintentionally overwritten here
+        await configAPI.update(payload);
+        refetchTeamConfig();
+      } else {
+        const payload = {
+          integrations: {
+            conditional_access_enabled: enableConditionalAccess,
+            // These fields will never actually be changed here. See comment above
+            // IGlobalIntegrations definition.
+            zendesk: teamConfig?.integrations.zendesk || [],
+            jira: teamConfig?.integrations.jira || [],
+          },
+        };
+        await teamsAPI.update(payload, teamIdForApi);
+        refetchGlobalConfig();
```

Comment:
why refetching global config here, and team config above?  seems backwards.

Reply:
Yep thanks for the catch, switching...

> so if we set this up for No Team then we add it to the global config

That's right! Not the clearest system...